test/e2e/runner.go

```diff
@@ -93,22 +93,32 @@ func main() {
 		if provision || cfg.ForceDeploy {
 			log.Printf("Soak cluster %s does not exist or has expired\n", rg)
 			log.Printf("Deleting Resource Group:%s\n", rg)
-			acct.DeleteGroup(rg, true)
+			if err = acct.DeleteGroup(rg, true); err != nil {
+				log.Fatalf("Unexpected error deleting Resource Group: %s", err)
```

Comment:
will this fail if the group doesn't exist?

Reply:
It will—it just calls `az group delete` under the covers, so if the resource group doesn't exist, the command returns 1 even with the `--no-wait` flag.

Maybe it's safer here to discard the error then:
`_ = acct.DeleteGroup(rg, true)`